internal/helper/const.go

```diff
@@ -3,10 +3,12 @@ package helper
 import "time"
 
 const (
-	OpenstackReconcileTime = 5 * time.Minute
-	DefaultRequeueTime     = 10 * time.Millisecond
-	RevisionAnnotation     = "loadbalancer.yawol.stackit.cloud/revision"
-	HashLabel              = "lbm-template-hash"
-	LoadBalancerKind       = "LoadBalancer"
-	VRRPInstanceName       = "ENVOY"
+	OpenstackReconcileTime   = 5 * time.Minute
+	DefaultRequeueTime       = 10 * time.Millisecond
+	RevisionAnnotation       = "loadbalancer.yawol.stackit.cloud/revision"
+	YawolKeepalivedFile      = "/tmp/yawolKeepalivedLastSet"
+	HashLabel                = "lbm-template-hash"
+	LoadBalancerKind         = "LoadBalancer"
+	VRRPInstanceName         = "ENVOY"
+	ContainsKeepalivedMaster = "ContainsKeepalivedMaster"
```

Comment:
Can we call the condition type `HasKeepalivedMaster`?
I think this is a more natural terminology when it comes to "sets".

If you agree, please rename all corresponding functions and variables accordingly.